EF/Services/SeansService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using EFVaiaa.DTOCinema;
+using EFVaiaa.EntitiesCinema;
+using System.Linq;
+using EFVaiaa.Interfaces;
+
+namespace EFVaiaa.Services
+{
+    class SeansService : ISeansService
+    {
+        readonly HallCRUDService hallCRUDService = new HallCRUDService();
```

Comment:
private readonly IHallCRUDService hallCRUDService;
public SeansService()
{
 this.hallCRUDService = new HallCRUDService();
}